app/src/main/java/edu/depaul/jmorton/boschhack/boschhackathon/accidents/Fire.java

```diff
@@ -0,0 +1,55 @@
+package edu.depaul.jmorton.boschhack.boschhackathon.accidents;
+
+import edu.depaul.jmorton.boschhack.boschhackathon.models.CarModel;
+import edu.depaul.jmorton.boschhack.boschhackathon.models.PersonModel;
+import edu.depaul.jmorton.boschhack.boschhackathon.models.TemperatureModel;
+
+/**
+ * Created by Dina on 9/20/16.
+ */
+
+public class Fire extends DangerousAccidents {
+    StringBuffer information = new StringBuffer();
```

Comment:
since getInformation has a return type StringBuffer, the return data will be stored in this buffer